<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputStrDialog</class>
 <widget class="QWidget" name="OutputStrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output Streams</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="Label10">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Output Stream</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QLabel" name="Label5">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="btnStream1">
     <property name="toolTip">
      <string>Select options for the first solution data stream</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8" colspan="3">
    <widget class="QComboBox" name="cBFormat1">
     <property name="toolTip">
      <string>Select time and position format for data stream</string>
     </property>
     <item>
      <property name="text">
       <string>Lat/Lon/Height</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X/Y/Z-ECEF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E/N/U-Baseline</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NMEA0183</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solution Status</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="btnStream2">
     <property name="toolTip">
      <string>Select options for the second solution data stream</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="8" colspan="3">
    <widget class="QComboBox" name="cBFormat2">
     <property name="toolTip">
      <string>Select time and position format for data stream</string>
     </property>
     <item>
      <property name="text">
       <string>Lat/Lon/Height</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>X/Y/Z-ECEF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E/N/U-Baseline</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NMEA0183</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Solution Status</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="lblF1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Output File Paths</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="10">
    <widget class="QLineEdit" name="lEFilePath1">
     <property name="toolTip">
      <string>Specify file name for first solution data stream</string>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QPushButton" name="btnFile1">
     <property name="toolTip">
      <string>Select file name for first solution data stream</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="10">
    <widget class="QLineEdit" name="lEFilePath2">
     <property name="toolTip">
      <string>Specify file name for second solution data stream</string>
     </property>
    </widget>
   </item>
   <item row="5" column="10">
    <widget class="QPushButton" name="btnFile2">
     <property name="toolTip">
      <string>Select file name for second solution data stream</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="cBTimeTag">
     <property name="toolTip">
      <string>Generate time-tags</string>
     </property>
     <property name="text">
      <string>Time-Tag</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="lblSwapInterval">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Swap Interval</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QComboBox" name="cBSwapInterval">
     <property name="toolTip">
      <string>Select swap interval for output data. Use specific keys (see &quot;?&quot; button) in the file path to generate unique filenames.</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLabel" name="lblH">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="btnKey">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="6" colspan="3">
    <widget class="QPushButton" name="btnOk">
     <property name="toolTip">
      <string>Accept changes</string>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="9" colspan="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="toolTip">
      <string>Cancle</string>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="Label6">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Stream Option</string>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="Label7">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="cBStream2C">
     <property name="toolTip">
      <string>Enable second output stream for solution data</string>
     </property>
     <property name="text">
      <string>(5) Solution 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QComboBox" name="cBStream2">
     <property name="toolTip">
      <string>Select stream type for second solution output stream</string>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Caster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="cBStream1C">
     <property name="toolTip">
      <string>Enable first output stream for solution data</string>
     </property>
     <property name="text">
      <string>(4) Solution 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="5">
    <widget class="QComboBox" name="cBStream1">
     <property name="toolTip">
      <string>Select stream type for first solution output stream</string>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Caster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
